### DISCUSSION TITLE
Feature request: Multi-species refinement

### DISCUSSION THREAD

**Discussion by alexjnoble** [nextpyp/.github#20] [Ideas]
Hello,

It would be useful to add multi-species particle refinement, as is done in Warp/M, for both the single particle and tomography workflows. This would be useful in cases where objects of interest are sparse, such as NPCs in cells, and where there are plenty of other non-objects of interest to co-refine, such as ribosomes in cells. Could this be added easily?

Thanks,
-Alex

**Comment by abartesaghi** [comment]
Thanks for the suggestion @alexjnoble. We haven't encountered any datasets where multi-species refinement would have made a big difference, but we do see the potential benefits of it. It is not straightforward to implement, but we may consider adding this feature going forward.
